ecosystem/sep-0018.md

```diff
@@ -4,7 +4,7 @@
 SEP: 0018
 Title: Data Entry Namespaces
 Author: MisterTicot (@misterticot), Leigh McCulloch (@leighmcculloch)
-Status: Active
+Status: Final
```

Comment:
SEP-18 contains a registry. If we mark it final will the registry still be updatable?

Reply:
I see two options,

1. SEPs in 'Final' cannot be updated in any form
2. SEPs in 'Final' can have patch-version updates

Personally I think the first option is the way to go, but that means we'd need to put the registry somewhere else.